tests/ulyanov_et_al_2016/test_modules.py

```diff
@@ -0,0 +1,136 @@
+import itertools
+
+import pytest
+
+import pytorch_testing_utils as ptu
+import torch
+from torch import nn
+
+from pystiche.image.utils import extract_num_channels
+from pystiche_papers.ulyanov_et_al_2016 import modules
+
+
+def test_SequentialWithOutChannels(subtests):
+    sequential_modules = [nn.Conv2d(3, 3, 1), nn.Conv2d(3, 3, 1)]
+    for out_channel_name in (
+        None,
+        1,
+        "1",
+    ):  # TODO: Rename out_channel_name is not implemented
```

Comment:
Could you clarify what you mean by this?

Reply:
Currently, any input other than this will result in a `KeyError`, because the key will not be renamed.